src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -0,0 +1,97 @@
+package teammates.test.cases.search;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackResponseCommentSearchResultBundle;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+
+/**
+ * SUT: {@link FeedbackResponseCommentsDb},
+ *      {@link teammates.storage.search.FeedbackResponseCommentDocument},
+ *      {@link teammates.storage.search.FeedbackResponseCommentSearchQuery}.
+ */
+public class FeedbackResponseCommentSearchTest extends BaseSearchTest {
```

Comment:
That kind of test has been existing since long ago. The spirit of this issue (and also #6474 in which I introduced the concept of search unit test) is that there was a glaring omission of unit tests in search-related back end logic, causing the burden of testing search-related stuffs to be shifted _entirely_ to UI tests, which is flawed.